src/screens/signup/signup2.tsx

```diff
@@ -1,32 +1,59 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { SafeAreaView, TouchableOpacity } from 'react-native';
 import { Ionicons } from '@expo/vector-icons';
 
 import { NavigationInterface } from '../types';
 import { useThemeContext } from '../../theme';
 
-import { Container, Welcome, Prompt, NavigationView, Pagination, OfText } from './styles';
+import Button from '../../components/button';
+import Input from '../../components/input';
+
+import {
+  Container,
+  Welcome,
+  Prompt,
+  NavigationView,
+  Pagination,
+  OfText
+} from './styles';
 
 interface SignupScreenTwoProp extends NavigationInterface {
   testID?: string;
 }
 
 export default function SignUpScreenTwo(props: SignupScreenTwoProp) {
   const { colors, fonts } = useThemeContext();
+
+  const [email, setEmail] = useState('clairetamara08@gmail.com');
+  const [values, setValues] =  useState({
+      firstName: '',
+      lastName: ''
+  })
+
   return (
     <Container>
-     <SafeAreaView>
-         <NavigationView>
-            <TouchableOpacity onPress={() => props.navigation.goBack()}>
+      <SafeAreaView>
+        <NavigationView>
+          <TouchableOpacity onPress={() => props.navigation.goBack()}>
             <Ionicons name="ios-arrow-back" size={35} color="#707070" />
-            </TouchableOpacity>
-            <Pagination>2 
-                <OfText> of </OfText>
-                 4</Pagination>
-         </NavigationView>
-         <Welcome>Good to have you</Welcome>
-         <Prompt>CREATE ACCOUNT</Prompt>
-     </SafeAreaView>
+          </TouchableOpacity>
+          <Pagination>
+            2<OfText> of </OfText>4
+          </Pagination>
+        </NavigationView>
+        <Welcome>Good to have you</Welcome>
+        <Prompt>CREATE ACCOUNT</Prompt>
```

Comment:
Use lower case letters and change them all to uppercase from CSS using text-transform uppercase